apps/web/src/app/events/[slug]/page.tsx

```diff
@@ -22,18 +22,43 @@ export async function generateMetadata({
   params: { slug: string };
 }): Promise<Metadata> {
   const event = await getEvent(params.slug);
+  const lectures = await getEventLectures(params.slug);
+
   if (!event) {
     notFound();
   }
+  lectures;
   let title = event.title;
   if (event.subtitle) {
     title += " " + event.subtitle;
   }
+
+  if (title.length > 55) {
+    title = title.substring(0, 55) + "...";
+  }
+
+  if (event.description.length > 100) {
+    event.description = event.description.substring(0, 100) + "...";
+  }
```

Comment:
tu można też jak z tytułem wyciągnąć to do zmiennej wyżej (`let description = event.description`)